DataDefinitions/InaraAPIEvent.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace EddiDataDefinitions
+{
+    public class InaraAPIEvent
```

Comment:
Hmm, not sure EddiDataDefinitions is the right home for this. I'd put in EddiInaraService I think.

Reply:
Updated. Tweaked to remove links to DataProviderService. Now only dependent on EDDI's core. New dependency graph:
![Dependencies Graph 3](https://user-images.githubusercontent.com/4382539/65096463-ee12f200-d979-11e9-873d-6f54cd673dbf.png)